provider/cpu_utils.py

```diff
@@ -130,3 +130,22 @@ def check_cpu_flags(params, flags, test, session=None):
         logging.info("Check cpu flags on host")
         if missing:
             test.cancel("This host doesn't support flag %s" % missing)
+
+
+def cpu_check_cmd_output(check_cmd, check_output, session=None):
```

Comment:
Same to https://github.com/autotest/tp-qemu/pull/2453.

Reply:
thanks and update.